snuba/util.py

```diff
@@ -127,17 +153,10 @@ def complex_column_expr(expr, body, depth=0):
         else:
             ret = ''
 
-    # emptyIfNull(col) is a simple pseudo function supported by Snuba that expands
-    # to the actual clickhouse function ifNull(col, '') Until we figure out the best
-    # way to disambiguate column names from string literals in complex functions.
-    if ret == 'emptyIfNull' and len(expr) >= 1 and isinstance(expr[0], tuple):
-        ret = 'ifNull'
-        expr = (expr[0] + ("''",),) + expr[1:]
-
     first = True
     for subexpr in expr:
         if isinstance(subexpr, tuple):
-            ret += '(' + complex_column_expr(subexpr, body, depth + 1) + ')'
+            ret = function_expr(ret, complex_column_expr(subexpr, body, depth + 1))
```

Comment:
This might be wrong in some cases.

Reply:
Can you elaborate?

Reply:
This was mostly a note I left myself last night to make myself look into it today. It might be ok actually, it passes tests, but I just wanted to convince myself that it was correct.